packages/csv-parser-discount-code/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools/csv-parser-discount-code",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Wasn't changed manually, but here 9b294a5c7ef3dda5d34c79a81aa5c7d9b5242bbb. Probably shows so due to the rebasing